arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-kenya.dts

```diff
@@ -319,9 +319,9 @@
 &i3c4 {
         status = "okay";
         initial-role = "primary";
-        bus-context = /bits/ 8 <I3C_BUS_CONTEXT_JESD403>;
-        i3c-scl-hz = <10000000>;
-        i2c-scl-hz = <1000000>;
+	internal-pullup = <2>;
```

Comment:
This commit also changes frequency , please update commit message with this detAils